lib/iex/lib/iex/helpers.ex

```diff
@@ -277,6 +279,106 @@ defmodule IEx.Helpers do
     end
   end
 
+  @module_info_keys [:module, :object_file, :source, :version, :compile_time, :compile_options]
+
+  @doc """
+  Prints out a list of all modules loaded and their BEAM file path.
+
```

Comment:
Sorry, I'm missing the point here. Is the problem @module_info_keys or the doc for m?